modules/programs/darcs.nix

```
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let cfg = config.programs.darcs;
+in {
+  meta.maintainers = with maintainers; [ chris-martin ];
+
+  options = {
+    programs.darcs = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "If enabled, install Darcs and config files.";
+      };
```

Use `mkEnableOption`

```suggestion
      enable = mkEnableOption "Darcs";
```